.github/workflows/container.yml

```
@@ -0,0 +1,42 @@
+          - "foxminchan/bookworm/notification"
+          - "foxminchan/bookworm/ordering"
+          - "foxminchan/bookworm/rating"
+          - "foxminchan/bookworm/chat"
+          - "foxminchan/bookworm/mcptools"
+    steps:
+      - name: Run Trivy on published containers
+        uses: aquasecurity/trivy-action@0.31.0
+        with:
+          image-ref: "ghcr.io/${{ matrix.container }}:latest"
+          format: "sarif"
+          output: "trivy-results-${{ matrix.container | replace('/', '-') }}.sarif"
```

actionlint (1.7.7)

got unexpected character ' ' while lexing || operator, expecting '|'

(expression)